src/lib/containers/table/table-top/ColumnSelection.tsx

```diff
@@ -30,34 +33,24 @@ export const ColumnSelection: React.FunctionComponent<ColumnSelectionProps> = (
     isColumnSelected,
     toggleColumnSelection,
     darkTheme,
-    facetAliases,
-    show: showFromProps,
-    onChange,
+    facetAliases
   } = props
 
-  let [show, setShow] = useState(false)
-
-  let [usedShow, usedSetShow] =
-    showFromProps !== undefined ? [showFromProps, onChange!] : [show, setShow]
```

Comment:
I think this was previously used by other components.  Now it's a subcomponent of TopLevelControls, which is a subcomponent of QueryWrapperPlotNav.  From what I see in the codebase, the show from props (showFromProps) is no longer set.